.expeditor/buildkite/cypress.sh

```diff
@@ -21,6 +29,6 @@ do
 
   if ! npm run cypress:record; then
       buildkite-agent artifact upload "cypress/videos/*;cypress/videos/**/*;cypress/screenshots/*;cypress/screenshots/**/*"
-      return 1
+      exit 1
```

Comment:
to make it more clear this is a cypress failure case, not a bash failure

Reply:
@blakestier thought the `return` made it look like the script itself had failed, rather than the cypress test.

Reply:
Indeed! 
```
$ grep . script_*.sh
script_one.sh:#!/bin/bash
script_one.sh:return 1
script_two.sh:#!/bin/bash
script_two.sh:exit 1
$ ./script_one.sh
./script_one.sh: line 3: return: can only `return' from a function or sourced script
$ echo $?
1
$ ./script_two.sh
$ echo $?
1
$
```